packages/components/src/internal/components/permissions/EffectiveRolesList.tsx

```diff
@@ -35,13 +40,35 @@ export class EffectiveRolesList extends React.PureComponent<Props, any> {
         return (
             <>
                 <hr className="principal-hr" />
-                <div className="principal-detail-label">Effective Roles</div>
-                <ul className="permissions-groups-ul">
-                    {assignments.map(assignment => {
-                        const role = rolesByUniqueName.get(assignment.role);
-                        return <li key={assignment.role}>{role ? role.displayName : assignment.role}</li>;
-                    })}
-                </ul>
+                <Row>
+                    <Col xs={4} className="principal-detail-label">
+                        Permissions
```

Comment:
We should continue to call this "Effective Roles". I think the spec didn't get updated after this discussion, and I know the tab is called "Permissions", but it is difficult to explain how roles work and are related to permissions if we use "Permissions" for everything. Also, we want to make it a little easier to know there's a difference between the roles explicitly assigned and those that are shown here.

Reply:
reverted. thanks for the explanation